src/structures/Chat.js

```diff
@@ -184,7 +184,7 @@ class Chat extends Base {
             if (searchOptions && searchOptions.limit > 0) {
                 while (msgs.length < searchOptions.limit) {
                     const loadedMessages = await chat.loadEarlierMsgs();
-                    if (!loadedMessages) break;
+                    if (!loadedMessages?.length) break;
```

Comment:
Please avoid using features that are not available in node v12 since we still support this version (optional chaining is not available)